src/styles/main.scss

```diff
@@ -1,3 +1,48 @@
-body {
-  margin: 0;
+@import "./colors.scss";
+@import "./header.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
+@mixin columns($column-count) {
+  grid-template-columns: repeat($column-count, 200px);
+}
+
+.page {
+
+  &__body {
+    margin: 0;
+    font-family: "roboto", sans-serif;
+  }
+
+  &__header {
+    padding: 0 50px;
+    box-sizing: border-box;
+  }
```

Comment:
Move it to header styles file

Reply:
this part relates to the page, not to the header.